xExtension-ReadingTime/static/readingtime.js

```diff
@@ -42,18 +42,20 @@
 
  flux_words_count: function flux_words_count(flux) {
 
-     reading_time.textContent = flux.textContent; // get textContent
+     reading_time.textContent = flux.childNodes[2].childNodes[1].textContent; // get textContent, from the article itself (not the header, not the bottom line). `childNodes[2].childNodes[1]` gives : `<div class="content no_limit">` element
```

Comment:
Seems a bit fragile. Would it be easier and less likely to change to do something like flux.querySelector('.content.no_limit')?

Reply:
That's right.
I tried that, but I couldn't get a result with that command (I don't understand why).
I could also just text the full text as before, then subtract an arbitrary value, the accuracy isn't such a big deal…

Reply:
Either way is fine by me, just trying to think along. ;-)

Reply:
I way thinking this way: that's a quick fix, still a bit better in accuracy than the "arbitrary value", it's a bit dirty but working (and I didn't really have the time to investigate further), if it breaks again (I'll see it, I'm using it) I'll change it to the arbitrary value (or have more time to properly do it).
I know, it's dirty ^^
I'm ok to change it if you want.